CharacterGenerator/CharacterGenerator/COD4database.m

```diff
@@ -0,0 +1,138 @@
+//
+//  COD4database.m
+//  CharacterGenerator
+//
+//  Created by Z on 8/2/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "COD4database.h"
+
+@implementation COD4database
+
+-(void)setup{
+    [self setupPrimaryWeapons];
+    [self setupSecondaryWeapons];
+    [self setupSpecialGrenades];
+    [self setupPerks];
+    [self setupTeam];
+}
+
+-(void)setupPrimaryWeapons{
+    
+    NSArray *assaultRifles = @[@"M16A4",
+                               @"M4A1",
+                               @"AK-47",
+                               @"G3",
+                               @"G36C",
+                               @"M14",
+                               @"MP44"
+                               ];
+    
+    NSArray *submachineGuns = @[@"MP5",
+                                @"Skorpion",
+                                @"Mini-Uzi",
+                                @"AK-74u",
+                                @"P90",
+                                @"MP44"
+                                ];
+    
+    NSArray *lightMachineGuns = @[@"M249 SAW",
+                                  @"RPD",
+                                  @"M60E4"];
+    
+    NSArray *shotguns = @[@"W1200",
+                          @"M1014"];
+    
+    NSArray *sniperRifles = @[@"M40A3",
+                              @"M21",
+                              @"Dragunov",
+                              @"R700",
+                              @"Barrett .50 Cal"
+                              ];
+    
+    NSDictionary *primaryWeapons = @{
```

Comment:
This might be more flexible and reusable if it was a `PrimaryWeapon` class that had a bunch of `NSArray` properties. And then all of this setup logic could go in the `init` method of the `PrimaryWeapon` class. Something like this:
```
PrimaryWeapons *primaryWeapons = [[PrimaryWeapons alloc] init];
soldier.primaryWeapon = [primaryWeapons.assaultRifles objectAtIndex:0];
```

Same for the secondary weapons. etc...